tests/containers/agencyLanding/AgencyLandingContainer-test.jsx

```diff
@@ -0,0 +1,154 @@
+/**
+ * AgencyLandingContainer-test.jsx
+ * Created by Lizzie Salita 7/18/17
+ */
+
+import React from 'react';
+import { mount, shallow } from 'enzyme';
+import sinon from 'sinon';
+import Immutable from 'immutable';
+import { Agency } from 'redux/reducers/agencyLanding/agencyLandingReducer';
+
+import { AgencyLandingContainer } from
+    'containers/agencyLanding/AgencyLandingContainer';
+import { AgencyLandingSearchBarContainer } from
+    'containers/agencyLanding/AgencyLandingSearchBarContainer';
+import AgencyLandingResultsSection from
+    'components/agencyLanding/AgencyLandingResultsSection';
+
+
+import { mockComponent, unmockComponent } from '../../testResources/mockComponent';
+import { mockData, mockMeta, mockAgenciesOrder } from './mockToptierAgencies';
+
+jest.mock('helpers/agencyLandingHelper', () => require('./agencyLandingHelper'));
+jasmine.DEFAULT_TIMEOUT_INTERVAL = 10000;
+
+const setup = (props) => shallow(<AgencyLandingContainer {...props} />);
+
+// spy on specific functions inside the component
+const fetchAgenciesSpy = sinon.spy(AgencyLandingContainer.prototype, 'fetchAgencies');
+const parseAgenciesSpy = sinon.spy(AgencyLandingContainer.prototype, 'parseAgencies');
+
+describe('AgencyLandingContainer', () => {
+    beforeAll(() => {
+        mockComponent(AgencyLandingResultsSection);
+        mockComponent(AgencyLandingSearchBarContainer);
+    });
+
+    it('should make an API request on mount', async () => {
+        // mount the container
+        const container = setup({
```

Comment:
Moved HTML from `AgencyLandingContainer` to new component `AgencyLandingContent`, mocked `AgencyLandingContent` in the test and now can use `mount` without getting an error.